pynxtools/testing/nexus_conversion.py

```diff
@@ -94,15 +95,15 @@ def convert_to_nexus(self, ignore_undocumented=False):
             template=Template(template), file_paths=tuple(input_files)
         )
 
+        entry_names = read_data.get_all_entry_names()  # type: ignore[attr-defined]
+        for entry_name in entry_names:
+            write_nexus_def_to_entry(read_data, entry_name, self.nxdl)
+
         assert isinstance(read_data, Template)
-        with self.caplog.at_level("ERROR", "WARNING"):
-            _ = validate_dict_against(
-                self.nxdl, read_data, ignore_undocumented=ignore_undocumented
-            )
-        assert not self.caplog.records, "Validation is not successful. Check logs."
-        for record in self.caplog.records:
-            if record.levelname == "ERROR":
-                assert False, record.message
+
+        with self.caplog.at_level(logging.WARNING):
```

Comment:
This introduces some minor changes to the assert, so that the plugin test is in line with how we test the core pynxtools in https://github.com/FAIRmat-NFDI/pynxtools/blob/71020b564b88e33a8e50770315b6cfa63ffea31e/tests/dataconverter/test_helpers.py#L529

Reply:
It does not grep any error only ẁarning` not the `error`. Isn't it good to replace the code on the `test_helper.py` not to skip any errors, or?

Reply:
It will also catch any errors. Any log messages with a severity level of logging.WARNING or higher (i.e., WARNING, ERROR, CRITICAL) will be captured by caplog if you use `with caplog.at_level(logging.WARNING)`. So, errors are already included at this level.